frontend/app/electron/main/index.ts

```diff
@@ -71,17 +71,42 @@ async function onReady(): Promise<void> {
 
   getWindow().webContents.on('before-input-event', (event, input) => {
     const win = getWindow();
-    if (isMac ? input.meta : input.control) {
-      if (['ArrowLeft', '['].includes(input.key) && win.webContents.canGoBack()) {
-        win.webContents.goBack();
-        event.preventDefault();
-      }
 
-      if (['ArrowRight', ']'].includes(input.key) && win.webContents.canGoForward()) {
-        win.webContents.goForward();
-        event.preventDefault();
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
Instead of suppressing the warning can we instead add a .catch that logs any error?
It should fix the problem and we should have some information in the logs if something goes wrong.